applications/serial_lte_modem/src/slm_at_socket.c

```diff
@@ -849,7 +847,7 @@ static int do_recvfrom(int timeout, int flags)
 	struct sockaddr remote;
 	socklen_t addrlen = sizeof(struct sockaddr);
 
-	ret = socket_poll(sock.fd, POLLIN, timeout);
+	ret = slm_sockopt_set(sock.fd, SO_RCVTIMEO, timeout);
```

Comment:
Do we now set the same timeout in every time we do recvfrom()?

Reply:
The timeout comes as a required AT-command parameter. So it can change with every call.